backend/app/services/inquiries.py

```diff
@@ -19,7 +19,9 @@ def get_inquiry_by_text(*, session: Session, text: str) -> Inquiry | None:
     return session_text
 
 
-def get_inquiry_by_id(*, session: Session, inquiry_id: uuid.UUID) -> Inquiry | None:
+def get_inquiry_by_id(*, session: Session, inquiry_id: UUID) -> Inquiry | None:
+    if not isinstance(inquiry_id, UUID):
```

Comment:
I'm not sure this check is necessary. Also we may migrate to a different class of identifier.

Reply:
Removing type check as suggested.